helpers/control.py

```diff
@@ -345,6 +345,9 @@ def tfw_log_file_exists() -> bool:
         """
         return remote.tempesta.exists(tf_cfg.cfg.get("TFW_Logger", "log_path"))
 
+    def load_module(self, path, module_name):
+        self._do_run(f"insmod {self.srcdir}/{path}/{module_name}.ko")
```

Comment:
`_do_run` method should use to start Tempesta FW.
```suggestion
        remote.tempesta.run_cmd(f"insmod {self.srcdir}/{path}/{module_name}.ko")
```